employee.rb

```diff
@@ -0,0 +1,28 @@
+class Employee
+  attr_accessor :name, :salary, :department, :review, :employee_raise, :satisfactory
+  def initialize(name, salary)
+    @salary=salary
+    @name=name
+    @department=Department.new(department)
+    @review = nil
```

Comment:
No need to set these to nil, since the default value for unset instance variables is nil anyway.